CITATION

```diff
@@ -1 +1,15 @@
-FIXME: describe how to cite this lesson.
\ No newline at end of file
+---
+layout: page
+title: "Citation"
+root: .
+---
+## Citation
+
+ESMValTool can be cited as:
+
+Righi, M., Andela, B., Eyring, V., Lauer, A., Predoi, V., Schlund, M., Vegas-Regidor, J., Bock, L., Brötz, B., de Mora, L., Diblen, F., Dreyer, L., Drost, N., Earnshaw, P., Hassler, B., Koldunov, N., Little, B., Loosveldt Tomas, S., and Zimmermann, K.: ESMValTool v2.0 – Technical overview, Geosci. Model Dev. Discuss., https://doi.org/10.5194/gmd-2019-226, in review, 2019.
+
+[Link](https://www.geosci-model-dev-discuss.net/gmd-2019-226/)
```

Comment:
This can be updated now that the paper has been published

Reply:
This citation is meant for the tutorial rather than the software and material. We need to describe how to cite the tutorial. It can be done when tutorial is published, for example in Zenodo and gets a DOI.